main.js

```diff
@@ -51,6 +70,7 @@ define(function (require, exports, module) {
     var _preferences,
         _command,
         _styleTag,
+        _styleInline,
         _Line,
         _LineGetHTML;
```

Comment:
Both, `_Line` and `_LineGetHTML` are unused variables in case you want to tidy that up.